samples/nrf_desktop/src/modules/button_remapper.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <kernel.h>
+#include <misc/util.h>
+
+#define MODULE button_remapper
+#include "module_state_event.h"
+#include "button_event.h"
+
+#include "hid_keymap.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_BOND_LOG_LEVEL);
+
+
+static bool fn_active;
+
+static u16_t remapped_key[32];
```

Comment:
Maybe make it as Kconfig option or #define instead of hardcoded value?

Reply:
fixed